package.json

```diff
@@ -94,6 +94,7 @@
     "request": "^2.88.0",
```

Comment:
Wouldn't we want to move request over to dev dependencies?

Reply:
Ideally 🙈. I wrote down that `gcs-resumable-upload` depends on it. Let me check.

Reply:
Should we maybe do that library first?